frontend/src/boot/i18n.ts

```diff
@@ -0,0 +1,24 @@
+import { boot } from 'quasar/wrappers';
+import messages from 'src/i18n';
+import Vue from 'vue';
+import VueI18n from 'vue-i18n';
+import Quasar from 'quasar';
+
+Vue.use(VueI18n);
+const locale = Quasar.lang.getLocale();
+
+export const i18n = new VueI18n({
+  locale: locale,
+  fallbackLocale: 'en-us',
+  messages,
+});
+
+export default boot(({ app }) => {
+  // Set i18n instance on app
+  app.i18n = i18n;
+});
+
+const params = new URLSearchParams(window.location.search);
```

Comment:
When I switch the locale manually in the URL, this is not reflected by what's shown in the language picker:

![umbra-locale-query-issue](https://user-images.githubusercontent.com/1479802/168116341-770fdfed-938d-40cc-b5f5-9c86f31e6291.gif)

I would think we'd want the presence of a query param to override and be reflected in the language picker component.